aviary/interface/utils/check_phase_info.py

```diff
@@ -198,10 +199,16 @@ def check_phase_info(phase_info, mission_method):
 
     # Check if all phases exist in phase_info
     for phase in phase_info:
-        if 'user_options' in phase_info[phase]:
-            phase_options = phase_info[phase]['user_options']
+        if mission_method is TWO_DEGREES_OF_FREEDOM:
+            base_phase = phase.removeprefix('reserve_')
+            if 'cruise' in base_phase:
+                base_phase = 'cruise'
+        else:
+            base_phase = phase
+        if 'user_options' in phase_info[base_phase]:
+            phase_options = phase_info[base_phase]['user_options']
```

Comment:
Is it possible to have both `target_duration` and `target_range`? If not, is there a check to raise an error if a user supplies both? Could you please add a check here if it doesn't exist?

Reply:
good idea